lib/mix/lib/mix/tasks/compile.app.ex

```diff
@@ -82,6 +82,8 @@ defmodule Mix.Tasks.Compile.App do
   ## Command line options
 
     * `--force` - forces compilation regardless of modification times
+    * `--compile-path path` - where to find `.beam` files and write the
```

Comment:
```suggestion
    * `--compile-path` - where to find `.beam` files and write the
```

I think that's the current format. Here are two examples: [`mix format`](https://github.com/elixir-lang/elixir/blob/479d5f9893b6537becb81a89f6a1d7cd8706d81c/lib/mix/lib/mix/tasks/format.ex#L66) and [`mix profile.cprof`](https://github.com/elixir-lang/elixir/blob/479d5f9893b6537becb81a89f6a1d7cd8706d81c/lib/mix/lib/mix/tasks/profile.cprof.ex#L34)